### PR TITLE
[DXKP-120] Reject requests without authorisation

### PR DESCRIPTION
**Pull request by carlosgaldino**
This patch adds support for checking the authorisation header and reject
any requests made with an incorrect token.

### CODE REVIEW

File: raftcat/src/kompass.rs [hunk 1]
```diff
@@ -12,7 +12,7 @@ use tokio::time;
 
 const REGISTER_PATH: &str = "register";
 const HEARTBEAT_PATH: &str = "heartbeat";
-const KOMPASS_AUTH_TOKEN: &str = "KOMPASS_AUTH_TOKEN";
+pub const KOMPASS_AUTH_TOKEN: &str = "KOMPASS_AUTH_TOKEN";
```
Comment (clux) [inline raftcat/src/kompass.rs:15]: not sure why you are defining a constant for this if it's only used once

Comment (carlosgaldino) [reply raftcat/src/kompass.rs:15]: It is also being used when sending the registration requests as well.

**Comment by carlosgaldino** [comment]
@clux Thanks for the review. Could you please merge? I get `You’re not authorized to merge this pull request.`

**Comment by clux** [comment]
Are you not in @dev-platform ? I added you guys to merge.

**Pull request merged**